app/domain/policies/failed.rb

```diff
@@ -1,7 +1,7 @@
 module Policies
   class Failed
     def self.call(scope)
-      scope.where(content_id: Audit.failing.select(:content_id))
+      scope.joins(:audit).merge(Audit.failing)
```

Comment:
How does ActiveRecord know that the join is on the `content_id`?

Reply:
https://github.com/alphagov/content-audit-tool/blob/52825161d79174aa2b6cb16060832f3c1b0144fa/app/models/audit.rb#L6-L7